modyn/trainer_server/internal/utils/training_info.py

```diff
@@ -71,4 +71,9 @@ def __init__(
         else:
             self.seed = None
 
+        if request.HasField("tokenizer"):
+            self.tokenizer = request.tokenizer.value
+        else:
+            self.tokenizer = ""
```

Comment:
Shall we use None instead of empty string?

Reply:
Usual problem of having GRPC in the middle. If a tokenizer with `request.tokenizer=None` is sent by the supervisor, it is received as `request.tokenizer=""` and consequently gives problems. If you prefer `None` I can use it only on the trainer server side and add something like
```
if self.tokenizer == "":
   selt.tokenizer = None
```

Reply:
Ok true. Changed to `Optional[str]`